packages/core-web/src/index.js

```diff
@@ -21,50 +21,61 @@ function scrollToUrlAnchorHeading() {
   }
 }
 
-function detectAndApplyFixedHeaderStyles() {
+function detectAndApplyHeaderStyles() {
   jQuery(':header').each((index, heading) => {
     if (heading.id) {
       jQuery(heading).removeAttr('id'); // to avoid duplicated id problem
     }
   });
 
-  const headerSelector = jQuery('header[fixed]');
-  const isFixed = headerSelector.length !== 0;
-  if (!isFixed) {
+  const headerSelector = jQuery('header[sticky]');
+  if (headerSelector.length === 0) {
     return;
   }
 
-  window.addEventListener('resize', () => {
```

Comment:
> Just a heads up, I think will have to update the fixed-header-padding height one more time during resize.

I've the "merged" this fix from the other PR into the resizeObserver changes below